packages/test/test-end-to-end-tests/src/test/benchmark/DocumentMatrixPlain.ts

```diff
@@ -79,7 +79,6 @@ const runtimeOptions: IContainerRuntimeOptions = {
 			state: "disabled",
 		},
 	},
-	gcOptions: { gcEnabled: false, disableGC: true, runGC: false },
```

Comment:
Haha wow GC is SO disabled here.

Reply:
The logic (before this change) was the following:
- In non-summarizer clients, GC was off.
- In summarizer clients, GC was on / off based on a condition.

I updated this change to do the same as above but via feature flags.